publications.rkt

```diff
@@ -510,10 +487,606 @@
+                 "J. Funct. Program"
+                 2006
+                 "http://dx.doi.org/10.1017/S0956796806006009")
+    (publication "Improving flow analyses via GammaCFA: abstract garbage collection and counting"
+                 "Matthew Might and Olin Shivers"
+                 "International Conference on Functional Programming (ICFP)"
+                 2006
+                 "http://doi.acm.org/10.1145/1159803.1159807")
+    (publication "Static analysis for syntax objects"
+                 "David Fisher and Olin Shivers"
+                 "International Conference on Functional Programming (ICFP)"
+                 2006
+                 "http://doi.acm.org/10.1145/1159803.1159817")
+    (publication "Continuations and transducer composition"
+                 "Olin Shivers and Matthew Might"
+                 "Programming Language Design and Implementation (PLDI)"
+                 2006
+                 "http://doi.acm.org/10.1145/1133981.1134016")
+    (publication "Environment analysis via Delta CFA"
+                 "Matthew Might and Olin Shivers"
+                 "Principles of Programming Languages (POPL)"
+                 2006
+                 "http://doi.acm.org/10.1145/1111037.1111049")))
+
+@(define mitch-pubs
+   (list
+    (publication "Romeo: a system for more flexible binding-safe programming"
+                 "Paul Stansifer and Mitchell Wand"
+                 "International Conference on Functional Programming (ICFP)"
+                 2014
+                 "http://doi.acm.org/10.1145/2628136.2628162")
+    (publication "A Resource Analysis of the π-calculus"
+                 "Aaron Joseph Turon and Mitchell Wand"
+                 "Electr. Notes Theor. Comput. Sci"
+                 2011
+                 "http://dx.doi.org/10.1016/j.entcs.2011.09.028")
+    (publication "Parsing reflective grammars"
+                 "Paul Stansifer and Mitchell Wand"
+                 "LDTA"
+                 2011
+                 "http://doi.acm.org/10.1145/1988783.1988793")
+    (publication "A separation logic for refining concurrent objects"
+                 "Aaron Joseph Turon and Mitchell Wand"
+                 "Principles of Programming Languages (POPL)"
+                 2011
+                 "http://doi.acm.org/10.1145/1926385.1926415")
+    (publication "Bottom-up beta-reduction: Uplinks and lambda-DAGs"
+                 "Olin Shivers and Mitchell Wand"
+                 "Fundam. Inform"
+                 2010
+                 "http://dx.doi.org/10.3233/FI-2010-328")
+    (publication "The Higher-Order Aggregate Update Problem"
+                 "Christos Dimoulas and Mitchell Wand"
+                 "VMCAI"
+                 2009
+                 "http://dx.doi.org/10.1007/978-3-540-93900-9_8")
+    (publication "Essentials of programming languages (3. ed.)"
+                 "Daniel P. Friedman and Mitchell Wand"
+                 "MIT Press"
+                 2008
+                 #f)
```

Comment:
Thanks for going through and looking up URLs for so many of these! I'll start adding them now.